src/lib.rs

```diff
@@ -51,7 +51,7 @@ pub use self::data_types::{unsafe_guid, Identify};
 pub use self::data_types::{CStr16, CStr8, Char16, Char8, Event, Guid, Handle};
 
 mod result;
-pub use self::result::{Completion, Result, ResultExt, Status};
+pub use self::result::{Result, ResultExt, Status};
```

Comment:
As mentioned in https://github.com/rust-osdev/uefi-rs/issues/360#issuecomment-1057738309 should we also be exporting `Error` here?